ERCS/erc-7626.md

```diff
@@ -0,0 +1,145 @@
+---
+eip: 7626
```

Comment:
```suggestion
eip: 7628
```
To avoid any perception of number sniping, I am assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.